primeSearchingUtilities.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#include<vector>
```

Comment:
Could you fix these to consistently have a space between the "#include" and the "<header>"?